src/posts/podcast-ep04-a-sneak-peek-into-upcloud-kubernetes-service.md

```diff
@@ -0,0 +1,432 @@
+---
+type: Blog
+title: A sneak peek into UpCloud Kubernetes Service
+subheading: In this episode of The Continuous & Cloud Podcast, Jacob and Lauri chat with Ville Törhönen, Product Owner at UpCloud, about UpCloud’s upcoming managed Kubernetes Service (UKS).
+authors:
+- jlarfors
+- lsuomalainen
+tags:
+- Kubernetes
+- Cloud
+date: 2022-08-16
+image: "/blogs/podcast-ep04-a-sneak-peek-into-upcloud-kubernetes-service.png"
+featured: true
+jobActive: true
+
+---
+<iframe title="Embedded podcast player" src="https://anchor.fm/verifa/embed/episodes/A-sneak-peek-into-UpCloud-Kubernetes-Service-e1miudr" height="151px" width="100%" frameborder="0" scrolling="no"></iframe>
+
+<div class="blog-flex">
+
+[![Listen on Spotify](/blogs/2021-03-30/listen-on-spotify.png)](https://open.spotify.com/show/12yStrneLdEsXn1Bjp6Myz)
+
+[![Listen on Apple Podcasts](/blogs/2021-03-30/listen-on-apple-podcasts.png)](https://podcasts.apple.com/gb/podcast/the-verifa-podcast/id1561051552)
+
+[![Listen on Google Podcasts](/blogs/2021-03-30/listen-on-google-podcasts.png)](https://www.google.com/podcasts?feed=aHR0cHM6Ly9hbmNob3IuZm0vcy81Mzg0NzE1Yy9wb2RjYXN0L3Jzcw==)
+
+</div>
+
+**Welcome to The Continuous & Cloud Podcast by Verifa, where we chat about continuous delivery, cloud architecture and most things inbetween.**
+
+In this episode of The Continuous & Cloud Podcast, Jacob and Lauri chat with Ville Törhönen, Product Owner at UpCloud, about UpCloud’s upcoming managed Kubernetes Service (UKS).
+
+## **During this episode we discuss**
```

Comment:
Titles shouldn't need the surrounding `**` because they are already at the correct weight. Do we do this for all other blogs, or where did this come from?

Reply:
I copied the format from a previous podcast page. So will remove them now.